packages/desktop-gui/src/lib/utils.js

```diff
@@ -85,13 +88,31 @@ module.exports = {
     }
   },
 
-  durationFormatted: (durationInMs) => {
+  durationFormatted: (durationInMs, padMinutes = true) => {
     const duration = moment.duration(durationInMs)
 
-    let durationHours = duration.hours() ? `${duration.hours()}h ` : ''
-    let durationMinutes = duration.minutes() ? `${duration.minutes()}m ` : ''
-    let durationSeconds = duration.seconds() ? `${duration.seconds()}s ` : ''
+    const durationSecs = duration.seconds() ? `${duration.seconds()}` : ''
+    const durationMins = duration.minutes() ? `${duration.minutes()}` : ''
+    const durationHrs = duration.hours() ? `${duration.hours()}` : ''
+
+    const total = _.compact([
+      durationHrs,
+      !!durationHrs || padMinutes ? padStart(durationMins, 2, '0') : durationMins,
```

Comment:
oh my, this expression is complicated

Reply:
It is! I mean, all I want is this logic in this spec to pass, so if there's a less complicated way, I'm game -> https://github.com/cypress-io/cypress/pull/1604/files#diff-6a3f8282b23f89f40bb2248f3e7d5c64